docs/callbacks/QUnit.begin.md

```diff
@@ -11,45 +11,62 @@ version_added: "1.0"
 
 `QUnit.begin( callback )`
 
-Register a callback to fire whenever the test suite begins. The callback can return a promise that will be waited for before the next callback is handled.
+Register a callback to fire whenever the test suite begins. The callback may be an async function, or a function that returns a promise, which will be waited for before the next callback is handled.
 
-`QUnit.begin()` is called once before running any tests.
+The begin callback will be called once before QUnit runs any tests.
```

Comment:
Styling: should it be monospaced `begin`? Getting in the weeds here...

Reply:
Rephrased.